From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Fri, 24 Feb 2023 18:13:08 -0500
Subject: [PATCH] loop-util: define LOOP_SET_BLOCK_SIZE for linux headers <
 4.14

---
 src/basic/missing_loop.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/basic/missing_loop.h b/src/basic/missing_loop.h
index b88501d78b..a3673e7656 100644
--- a/src/basic/missing_loop.h
+++ b/src/basic/missing_loop.h
@@ -26,6 +26,10 @@ assert_cc(LO_FLAGS_DIRECT_IO == 16);
 assert_cc(LOOP_SET_DIRECT_IO == 0x4C08);
 #endif
 
+#ifndef LOOP_SET_BLOCK_SIZE
+#define LOOP_SET_BLOCK_SIZE 0x4C09
+#endif
+
 #ifndef LOOP_SET_STATUS_SETTABLE_FLAGS
 #  define LOOP_SET_STATUS_SETTABLE_FLAGS (LO_FLAGS_AUTOCLEAR | LO_FLAGS_PARTSCAN)
 #endif
-- 
2.34.1

