From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Wed, 10 Sep 2025 11:44:41 -0400
Subject: [PATCH] nspawn-seccomp: Include missing_capability.h so CAP_BPF is
 defined

---
 src/nspawn/nspawn-seccomp.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nspawn/nspawn-seccomp.c b/src/nspawn/nspawn-seccomp.c
index 6956689a..b05d91b4 100644
--- a/src/nspawn/nspawn-seccomp.c
+++ b/src/nspawn/nspawn-seccomp.c
@@ -5,6 +5,7 @@
 #include <sys/socket.h>
 
 #include "log.h"
+#include "missing_capability.h"
 #include "nspawn-seccomp.h"
 #include "seccomp-util.h"
 #include "strv.h"
-- 
2.34.1

